jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+
+export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalrun" -c "base ocnanal ocnanalrun"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+export COMPONENT="ocean"
```

Comment:
Is it necessary to define and use this variable.
It is clear from the name of this script, this is an `OCEAN` component.

Reply:
revision committed